### PR TITLE
Lint all Svelte files

### PR DESCRIPTION
**Pull request by stevekinney**
Due to a previous misconfiguration, we were not using the Svelte plugin for ESLint on our `.svelte` components. Previously, we add all of Holocene and addressed the issues. This change adds _all_ `.svelte` files and address any issues found in those files.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
Comment (rossedfort) [inline pnpm-lock.yaml:1]: May need to confirm this change with team, we usually try to upgrade all at the same time

**Review by rossedfort — APPROVED** [review]
Looks good pending the merge conflicts, lmk if you want a second pair of eyes on those 👀

File: src/lib/holocene/combobox/combobox.svelte [hunk 2]
```diff
@@ -146,8 +151,12 @@
       return option;
     }
 
-    if (isObjectOption(option) && canRenderCustomOption(option)) {
-      return option[optionLabelKey];
+    if (
+      typeof option === 'object' &&
```
[the remark this answers is not in the document]
Comment (stevekinney) [reply src/lib/holocene/combobox/combobox.svelte:155]: Tell that to TypeScript.

[the remark this answers is not in the document]
Comment (stevekinney) [reply src/lib/holocene/combobox/combobox.svelte:155]: Actually, I figured it out by adding `option is T & object` to the check.

**Pull request merged**